packages/select/Option.tsx

```diff
@@ -0,0 +1,63 @@
+import { h, Component } from 'skatejs';
+import { GenericEvents } from '../_helpers/index';
+import { prop, customElement, shadyCssStyles } from '../_helpers/decorators';
+import { bind } from 'decko';
+import { SelectCardItem } from './components/CardItem';
+import styles from './Select.scss';
+
+export type OptionProps = Props & EventHandlers;
+
+export type Props = {
+  value?: string,
+};
+
+export type Events = {
+  click?: GenericEvents.ClickHandler,
+};
+export type EventHandlers = {
+  onClick?: GenericEvents.ClickHandler,
+};
+
+@shadyCssStyles()
+@customElement( 'bl-option' )
+export default class Option extends Component<OptionProps> {
+
+  @prop( { type: Boolean } ) disabled: boolean;
+  @prop( { type: Object } ) value: any;
+  @prop( { type: Boolean } ) _selected = false;
```

Comment:
underscores for class properties are used only if you create getter/setter to that property ( which you do )
pls explicitly use private `@prop( { type: Boolean } ) private _selected: false`,